libsignal-service/src/push_service.rs

```diff
@@ -592,6 +592,23 @@ pub trait PushService: MaybeSend {
         }
     }
 
+    async fn get_sticker_pack_manifest(
+        &mut self,
+        id: &str
```

Comment:
```suggestion
        id: &str,
```

rustfmt has two comments